connect/file/src/test/java/org/apache/kafka/connect/file/FileStreamSourceTaskTest.java

```diff
@@ -73,7 +75,6 @@ public void teardown() {
     }
 
     private void replay() {
```

Comment:
I wonder if it would be simpler to get rid of the `replay()` method and just call `verifyAll()` in the tests that need it. WDYT?

Reply:
Yeah, we have been stuck into the old code, this is the simplest way. 😂, Done!